migrations.lock

```diff
@@ -145,6 +147,10 @@ auth
  0010_alter_group_name_max_length
  0011_update_proxy_permissions
  0012_alter_user_first_name_max_length
+authtoken
+ 0001_initial
+ 0002_auto_20160226_1747
+ 0003_tokenproxy
```

Comment:
Are these needed? I don't see any usages of DRF AuthTokens

Reply:
Custom auth class is inheriting from `TokenAuthentication` and overrides `authenticate` method. This is then called by the framework as part of decorator `@authentication_classes((ABDMUserAuthentication,))` in the views.